### PR TITLE
Fix Dockerfile.sublime-syntax

### PR DESCRIPTION
**Pull request by snxx-lppxx**
Hi @asbjornenge, please review this.

- Screenshots: <https://github.com/snxx-lppxx/dockerfile-sublime/blob/master/screenshots/dark-theme.png>.
- Test: <https://github.com/snxx-lppxx/dockerfile-sublime/blob/master/test/syntax_test.Dockerfile>.
---
**Docker.tmbundle:**
image

### CODE REVIEW

**Comment by asbjornenge** [comment]
Again, I have not worked on this repo for years and do not really know what is what anymore 😅 Maybe @princemaple can help? If not, I'm willing to add more maintainers 👍

File: Syntaxes/Dockerfile.sublime-syntax [hunk 1]
```diff
@@ -1,114 +1,57 @@
...
-      captures:
-        1: punctuation.whitespace.comment.leading.dockerfile
-        2: comment.dockerfile
-        3: punctuation.definition.comment.dockerfile
-
-  invalid:
-    - match: ^[^A-Z\n](.*)$
-      scope: invalid
-      set: body
+      scope: punctuation.definition.string.begin.toit
```
Comment (princemaple) [inline Syntaxes/Dockerfile.sublime-syntax:34]: what's `toit`?

File: Syntaxes/Dockerfile.sublime-syntax [hunk 2]
```diff
@@ -1,114 +1,57 @@
...
+    - include: operator-word
+    - include: const
+    - include: string-double
+
+  keyword:
+    - match: '^(FROM|CMD|COPY|WORKDIR|RUN|ENV|ADD|LABEL|STOPSIGNAL|USER|VOLUME|ONBUILD|EXPOSE|MAINTAINER|ARG|ENTRYPOINT|SHELL)'
+      scope: keyword.control.Dockerfile
+
+  operator-word:
+    - match: \b(apt\-get|get|install|clean|rm\s*\-rf|pip3|pip|python3|python|echo|exit)\b
```
Comment (princemaple) [inline Syntaxes/Dockerfile.sublime-syntax:29]: I think this is too specific. This PR would be great to be merged with https://github.com/asbjornenge/Docker.tmbundle/pull/24

**Review by princemaple — CHANGES_REQUESTED** [review]

**Comment by princemaple** [comment]
Actually, #24 looks really good, and seems to be a superset of this PR.

**Pull request closed without merging**